Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml

```diff
@@ -25,6 +25,7 @@
     ClothingNeckScarfStripedAce: 3
     ClothingNeckScarfStripedAro: 3
     ClothingNeckScarfStripedBiSexual: 3
+    ClothingNeckScarfStripedGay: 3
```

Comment:
Please put a comment indicating this is a CD change

Reply:
that is marked as a CD change

Reply:
While you are looking at this, could you add a comment where the CD content ends?

Reply:
Done it